Update text and add Ops tables.

ktlim opened the pull request:
- Change from FLOPS to core-hours for compute.
- Include Qserv storage and compute explicitly.
- Include Chilean DAC.
- Update HSC PDR2 to actual size instead of estimate.
- Add comparison with DES.
- Add Ops tables and associated text.

ktlim committed:
Update sizing model text.

Change from FLOPS to core-hours for compute.
Include Qserv storage and compute explicitly.
Include Chilean DAC.
Update HSC PDR2 to actual size instead of estimate.
Add comparison with DES.

ktlim committed:
Re-extract tables.

ktlim committed:
Add text for Ops tables.

ktlim committed:
Change Ops tables to landscape.

ktlim committed:
Bold first row of table.

womullan commented on Inputs.tex:
```diff
@@ -2,24 +2,34 @@
-{FLOPs Needed Total (DRP)}&{}&{2.37E+20}&{2.37E+20}&{2.66E+21}&{1.48E+22} \\ \hline
-{Annual Increase}&{}&{2.37E+20}&{0.00E+00}&{2.42E+21}&{1.21E+22} \\ \hline
+{Core-hours Needed Total (DRP)}&{}&{4.41E+06}&{4.41E+06}&{2.06E+07}&{1.02E+08} \\ \hline
+{Annual Increase}&{}&{4.41E+06}&{0.00E+00}&{1.62E+07}&{8.11E+07} \\ \hline
```
In the switch to core hours do the number of hours drop by the presumed faster processors available ?

ktlim replied:
No, they do not.  These are all "nominal" cores; any technological advances need to be taken into account in the costing.

ktlim replied:
I say explicitly: "Note that the nominal CPU does not evolve with time; if future CPUs do more work per core, the actual core-hours may be less than estimated here."

womullan commented on makeTablesFromGoogle.py:
```diff
@@ -139,11 +140,15 @@ def genTables(values):
+                bold_next = True
             else:
                 if name and row:
-                    if row[0].startswith('Year') or row[0].startswith('Total'):
```
thanks I was thinking of doing that !

womullan commented on datasetSizingOps.tex:
```diff
@@ -0,0 +1,16 @@
+\tiny \begin{longtable} { |p{0.22\textwidth}  |r  |r  |r  |r  |r  |r  |r  |r  |r  |r  |r  |r  |r |} 
+\caption{Dataset sizes used to calculate storage needs during Operations \label{tab:datasetSizingOps}}\\ 
+\hline 
+\textbf{Dataset Sizing}&\textbf{unit}&\textbf{LOY1/ FY23}&\textbf{LOY2/ FY24}&\textbf{LOY3/ FY25}&\textbf{LOY4/ FY26}&\textbf{LOY5/ FY27}&\textbf{LOY6/ FY28}&\textbf{LOY7/ FY29}&\textbf{LOY8/ FY30}&\textbf{LOY9/ FY31}&\textbf{LOY10/ FY32}&\textbf{Notes} \\ \hline
```
I suggest not putting the notes int he table int the doc - rather put them in the text this will make the table fit at least landscape from what i see.  I had left notes out on some other tables also . They remain in the sheet just not in the doc.  This applies to all the ops tables.

ktlim replied:
I removed the notes, adjusting the number of columns in the spreadsheet and the range extracted in the Makefile.

ktlim committed:
Add auto-extraction of tables.

ktlim committed:
Update tables.

ktlim committed:
Corrections to model.

Raw images go to object store directly.
Qserv nodes are estimated rather than disk and cores.
Compressed image sizes are used to estimate DRP compute.
Tape holds all the raw images and each data product in each DR.

ktlim committed:
Add overheadCost table and update others.
